DXRModules/DeusEx/Classes/DXRStats.uc

```diff
@@ -565,11 +576,20 @@ function AddDXRCredits(CreditsWindow cw)
     kos = dxr.flagbase.GetInt('DXRStats_knockouts');
     killsByOther = dxr.flagbase.GetInt('DXRStats_kills_by_other');
     kosByOther = dxr.flagbase.GetInt('DXRStats_knockouts_by_other');
+    mapcoverage = dxr.flagbase.GetInt('DXRStats_mapcoverage');
     deaths = GetDataStorageStat(dxr, "DXRStats_deaths");
     saves = player().saveCount;
     autosaves = GetDataStorageStat(dxr, "DXRStats_autosaves");
     loads = GetDataStorageStat(dxr, "DXRStats_loads");
 
+    //Calculate percentage of maps visited
+    if(#defined(revision)){
```

Comment:
should check if Revision is using vanilla maps too

Reply:
Corrected in 46a29ab